### PR TITLE
Synchronised/updated systers fork of hyperkitty with original project in mailmain repo

### LINKED ISSUE #12: Update Hyperkitty to reflect updates in original repo.
## Description
Hyperkitty has changed a lot since it was last cloned in 2015, updates need to be made to reflect updates at [https://gitlab.com/mailman/hyperkitty/](https://gitlab.com/mailman/hyperkitty/)

## Acceptance Criteria

- [x] Make require update

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
4 hours

### PR DESCRIPTION
**Pull request by Fenn-CS**
### Description
Updated project to reflect hundred of updates added over the past 3 years.

Fixes #12

### Type of Change:
- Code
- Quality Assurance
- User Interface
- Documentation

### How Has This Been Tested?
Tests pass

### Checklist:
**Delete irrelevant options.**
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,6 +4,21 @@
 HyperKitty - Archiver for GNU Mailman v3
 ========================================
 
+.. image:: https://gitlab.com/mailman/hyperkitty/badges/master/build.svg
```
Comment (Monal5031) [inline README.rst]: Shouldn't this  README reflect systers?

Comment (Fenn-CS) [reply README.rst]: That sounds off-track, the concern is first to **synchronized** the repository. Moreover you should check if the [previous documentation](https://github.com/systers/hyperkitty/blob/develop/README.rst) "reflected systers"?  
I had already created an [issue to localize documentation](https://github.com/systers/hyperkitty/issues/14)

Comment (Monal5031) [reply README.rst]: Oh! my bad didn't check it. LGTM then :+1:

Comment (Fenn-CS) [reply README.rst]: No issues :)

**Pull request merged**